app/src/main/java/de/westnordost/streetcomplete/data/osmnotes/OsmNoteQuestDao.java

```diff
@@ -110,6 +116,11 @@
 			values.put(Columns.COMMENT, quest.getComment());
 		}
 
+		if (quest.getImagePaths() != null)
+		{
+			values.put(Columns.IMAGE_PATHS, quest.getImagePaths().toString());
```

Comment:
Serialization issue. See the other comments.

Reply:
Done